src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -109,10 +110,24 @@ class BasePaymentsPage extends React.Component {
      */
     setPositionAddPaymentMenu(position) {
         this.setState({
-            anchorPositionTop: position.bottom,
+            anchorPositionTop: position.top + position.height,
 
             // We want the position to be 13px to the right of the left border
-            anchorPositionLeft: position.left + 13,
+            anchorPositionRight: (window.innerWidth - position.right) + 13,
+        });
+    }
+
+    /**
+     * Set position of the payment method menu
+     *
+     * @param {Object} position
+     */
+    setPositionAddPaymentMethod(position) {
+        this.setState({
+            anchorPositionBottom: window.innerHeight - position.top,
```

Comment:
:scream:

Reply:
Any change you suggest? should I avoid window.innerHeight and look for some alternative?

Reply:
You can use withWindowDimensions HOC to get these.

Reply:
Oh sure, thank you very much. I'll make the change

Reply:
Change made